inference-engine/src/gna_plugin/layers/gna_layer_info.hpp

```diff
@@ -194,7 +195,43 @@ class LayerInfo {
     bool isPermute() const noexcept {
         return isOfType("permute");
     }
-    bool isPooling() const noexcept {
+        // @brief this not only mathematically trivial, has some WA for kaldi case
+        bool isTrivialPermute() {
```

Comment:
It looks like formatting here is not correct. @admitriev-gna Could you please double check?